### PR TITLE
Homematic: Some updates

### PR DESCRIPTION
**Pull request by gerrieg**

### CODE REVIEW

**Comment by gerrieg** [comment]
@kaikreuzer 
Any chance to get this merged? This PR contains many fixes and improvements.

**Comment by kaikreuzer** [comment]
Uff, that's a huge one. We are in code freeze for 2.0 already - is this something really critical to be fixed? And is is thoroughly tested so that it for sure does not break things?

**Comment by gerrieg** [comment]
As always i do my best to support all these different devices a find workarounds for the CCU bugs. But i can only test with my own devices.
The Rainsensor for example only works with this PR. That's the only device of which i know that does not work. 

Because of the code freeze maybe it's better to release 2.0 as it is and push a 2.0.1 with these changes soon. So there is a chance for others to test with the 2.0.1 snapshot builds.

**Comment by kaikreuzer** [comment]
Sounds like a plan! I'd feel safer with this approach as well.

File: addons/binding/org.openhab.binding.homematic/ESH-INF/thing/bridge.xml [hunk 1]
```diff
@@ -91,6 +91,12 @@
 				<advanced>true</advanced>
 				<default>8701</default>
 			</parameter>
+            <parameter name="groupPort" type="integer">
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.homematic/ESH-INF/thing/bridge.xml]: could you run the code formatter?

File: addons/binding/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/communicator/AbstractHomematicGateway.java [hunk 2]
```diff
@@ -155,17 +165,19 @@ public void initialize() throws IOException {
         if (sb.length() > 2) {
             sb.setLength(sb.length() - 2);
         }
-        logger.info("Used Homematic transfer modes: " + sb.toString());
+        logger.debug("Used Homematic transfer modes: " + sb.toString());
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/communicator/AbstractHomematicGateway.java]: please use parameterized logging

**Comment by gerrieg** [comment]
yes, please merge it, thank you!

**Pull request merged**